### PR TITLE
Add msitools to docker beats golang-crossbuild 1.17 main image

### PR DESCRIPTION
**Pull request by aleksmaus**
Need to be able to extract the a file from .MSI distro for osquerybeat. 
The current workaround is installing msitools during osquerybeat build time for window platform.
Including this into the builder image would allow us to remove the workaround.

Closes: https://github.com/elastic/beats/issues/29826

Screenshot verifying the local image build msiextract tool is installed with this change:
Screen Shot 2022-01-24 at 9 21 57 AM

### CODE REVIEW

**Review by kuisathaverat — APPROVED** [review]
LGTM as soon as the CI pass it can be merged

**Comment by aleksmaus** [comment]
@v1v @kuisathaverat 
I don't have write access to the repository, can't merge.

**Pull request merged**
